tests/metrics/genai/prompts/test_v1.py

```diff
@@ -49,30 +49,38 @@ def test_evaluation_model_output():
 
     grading_context = {"ground_truth": "This is an output"}
     args_string = "\n".join(
-        [f"Provided {arg}: {arg_value}" for arg, arg_value in grading_context.items()]
+        [f"key: {arg}\nvalue:{arg_value}" for arg, arg_value in grading_context.items()]
     )
     expected_prompt1 = """
-    Please act as an impartial judge and evaluate the quality of the provided output which
-    attempts to produce output for the provided input based on a provided information.
-    You'll be given a grading format below which you'll call for each provided information,
-    input and provided output to submit your justification and score to compute the correctness of
-    the output.
+    Task
+    You are an impartial judge. You will be given an input that was sent to a machine
+    learning model, and you will be given an output that the model produced. You
+    will also be given additional information that was used by the model to generate the output.
 
-    Input:
+    Your task is to determine a numerical score called correctness based on the input and output.
+    A definition of correctness and a grading rubric are provided below.
+    You must use the grading rubric to determine your score. You must also justify your score.
+
+    Examples are included below for reference. Please make sure to use them as references and to
+    understand them before completing the task.
+
+    Input
     This is an input
 
-    Provided output:
+    Output
```

Comment:
Concerned about the lack of separators between section titles and content. As a human, it's hard to read,
so, as an LLM, presumably it's hard to read?

@suanmiao Do you have evidence here?

At least colons would make sense. I would think that underlines or markdown would be good.

Reply:
+1, having a colon would be helpful, I just don't recommend using "===============" since it costs more tokens and also is not as friendly to LLM as comparing to new lines or colon since repeated symbols might cause the model to repeat (this is the case for smaller model, but not necessarily for bigger models).